samples/monster_generated.h

```diff
@@ -510,6 +511,14 @@ inline bool VerifyEquipment(flatbuffers::Verifier &verifier, const void *obj, Eq
   }
 }
 
+inline bool VerifyEquipmentVector(flatbuffers::Verifier &verifier, const flatbuffers::Vector<flatbuffers::Offset<void>> *values, const flatbuffers::Vector<uint8_t> *types) {
```

Comment:
actually the sample schema is not changed. the code here generated is due to the change of the GenUnionPost, which is invoked on the existing union.